statsmodels/genmod/tests/results/results_glm.py

```diff
@@ -684,9 +691,15 @@ def __init__(self):
         # data set up for data not in datasets
         filename = os.path.join(os.path.dirname(os.path.abspath(__file__)),
             "stata_lbw_glm.csv")
-        data=np.recfromcsv(open(filename, 'rb'))
-        vfunc = np.vectorize(lambda x: x.strip(asbytes("\"")))
-        data['race'] = vfunc(data['race'])
+
+        # https://github.com/statsmodels/statsmodels/pull/4432#issuecomment-379279617
+        if NUMPY_LT_113 or PY2:
+            data=np.recfromcsv(open(filename, 'rb'))
+            vfunc = np.vectorize(lambda x: x.strip(asbytes("\"")))
+            data['race'] = vfunc(data['race'])
+        else:
+            data = pd.read_csv(filename).to_records()
```

Comment:
Depending on what the goal is here, consider passing `float_precision='high'` to pd.read_csv, and/or `index=False` to `to_records`.